Makefile

```diff
@@ -0,0 +1,53 @@
+SOURCES = main.py
```

Comment:
Add required files here in `SOURCES` or in `SUPPORT` variables which are essential to execution
something like.
```
SOURCES = main.py nepali_calendar.py preference.py convert.py
```
like you `date-converter.py` and `preference.py` files too..
:no_good: forgot to include this, I don't want to re-clone :sheep:.